roles/ceph-facts/tasks/facts.yml

```diff
@@ -92,54 +97,35 @@
     - not rolling_update
     - ceph_current_status.rc == 0
 
-- name: set_fact fsid from ceph_current_status
-  set_fact:
-    fsid: "{{ ceph_current_status.fsid }}"
+- block:
+  - name: set_fact fsid from ceph_current_status
+    set_fact:
+      fsid: "{{ ceph_current_status.fsid }}"
+
+  - name: reuse cluster fsid when cluster is already running
```

Comment:
I wonder if we actually need to write this down since the fsid is a fact and given it's never re-used.

Reply:
looks like we don't need it